modules/indigo/module/inc/indigo/port_manager.h

```diff
@@ -225,6 +225,22 @@ extern indigo_error_t indigo_port_queue_config_get(
     of_queue_get_config_reply_t **queue_config_reply);
 
 
+/**
+ * @brief Process an OF queue stats request handler
+ * @param queue_stats_request The LOXI request message
+ * @param cxn_id Connection ID on which the message arrived
+ * @return Return code from operation
```

Comment:
Should there be some english such as "The handler should return INDIGO_ERROR_NONE if the request has been successfully handled; otherwise an error reply will be sent with type XXX and code YYY" ?

Reply:
Will let the handler() reply the error code to the controller. The return code here will be casted to void and ignored by indigo.

Reply:
ok, if the handler returns the error code to the controller, then the prototype for the handler should have "void" as its return type.